src/main/services/CaseService.ts

```diff
@@ -77,12 +77,25 @@ export class CaseApi {
       },
     });
   };
+
+  updateDraftCase = async (caseType: string, caseId: string, formData: string): Promise<CaseDraftResponse> => {
```

Comment:
Instead of three parameters, lets just pass "req.session.userCase". CaseType, Id and the dob will all be on this object